### DISCUSSION TITLE
[error] commissioner start Error 35: InvalidCommand

### DISCUSSION THREAD

**Discussion by Glenyz** [openthread/openthread#10413] [Q&A]
I can't send this command to my FTD Commissioner. I am following [this codelab](https://openthread.io/codelabs/openthread-hardware#8) and I can't get step 9 to work. 

So far I've followed the steps up to before step 9 and the outputs of each step match with the codelab examples.

I tried adding the ot-ctl executable to my system $PATH, but to no avail.

The output is exactly as follows:

```
>commissioner start 
Error 35: InvalidCommand
``` 

Additionally I looked at the openthread [cli command reference](https://openthread.io/reference/cli/commands#commissioner_start) page and found saw I need to set ```OPENTHREAD_CONFIG_COMMISSIONER_ENABLE``` and ```OPENTHREAD_FTD``` but I have no clue how to do that. 

I'm working in Linux on a terminal based system if that affects anything.
If anyone knows how to set this command up that would be of tremendous help, thanks.

**Comment by jwhui** [comment]
Use the `OT_COMMISSIONER` build flag to enable the commissioner feature.

See the following build command detailed in: https://openthread.io/codelabs/openthread-hardware#build-and-flash_1
```
script/build nrf52840 USB_trans -DOT_JOINER=ON -DOT_COMMISSIONER=ON
```
